### ISSUE TITLE
UX Discussion: LOTR Project, NYPL Menu, ORBIS

### ISSUE THREAD

**Issue by ebeshero** [ebeshero/DHClass-Hub#265]
Choose one of the following digital archives to explore, and write a post in this thread discussing: 
1) how visual aids are used to convey information on the site, and how well (or badly) do these work? What seems unclear to you, and/or what seems to work really well? 
2) how effective is the (“UX”) in navigating at least one or two significant areas of the site? 
(Each of you should begin by writing a "UX review" of a site, and then read and respond to each other by looking at the sites others chose.) 

* [LOTR Project: Visualizing Tolkien’s Works on the Web](http://lotrproject.com/)
* [What’s on the menu?](http://menus.nypl.org/) (be sure to read the About page for the project goals)
* [ORBIS](http://orbis.stanford.edu/): The Stanford Geospatial Network Model of the Roman World

**Comment by Samantha-Mcguigan** [comment]
I chose to look at the site, "What's on the Menu." When first navigating the site it is a little confusing as to what the goal of the site is, but after reading the "about page" it becomes clear that it is a site where people work together to transcribe restaurant menus throughout history. The visual aids on the initial page show small squares with a preview of what menu it is and when you scroll over it it moves down to show the rest of the menu. I like this idea but they are kind of hard to see so I would prefer if they were slightly larger. I also think the layout of the page is good at showing all the different aspects of the site but I think how it is displayed could be more clear. For example, the different sections such as the "Frutti di Mare" and the "Map our Menus" seem to run into each other. I think it would be easier to navigate the site if there was more space between the different sections. I can also see this in the sections underneath because even though they are two different sections, the images of the menus have the same space between them as the images in the one section above them. I would also prefer a larger map on the "Map our Menus" page of the site so you can see the places where the menus were created more clearly and get a more accurate location when you zoom in. One thing I do like however, is that when you see the full image of the menus they are the actual menus themselves and the photos are very clear and easy to read. 

The "Map our Menus" section of the site is easy for a user to navigate. It is on the main page and when you click on the image of the map or the "Geotagger" tool highlighted in green, it takes you right to the page that has everything you need to find the place on a map where a menu was created. The only thing that is strange to me it that every time you click on it, it brings up a new menu and if you want to work with a different one, you have to go to the page where there is a list of menus, click on the menu you want, and then click on the green link in the corner that says "Map this Menu." I think it would be better if there was an area on the "Map the Menu" page to search for other menus or a link to the page of the site that has the list of menus. I do like that they have a link under the map that says "view menu record page" that allows you to see more information of the menu you are mapping.
The other section of the site that I want to review is the "Menus" page where you view all of the different menus the site has. I think it is easy to navigate this page because there are several different ways to search the menus such as by date, name, or dish count. You can also use the search engine in the top right corner to search through the menus by keyword. Also, having the options to show all, new, or menus that have already been transcribed is very useful. I think it would also be useful to be able to search the menus by location or language. You can also get back to the main page of the site by clicking on the words "What's on the Menu" in the top right corner but a user may not know this so I think it might be easier to navigate this if the words changed color or something when you scroll over them.

**Comment by bsf15** [comment]
I also chose to look at the site “What’s on the Menu?” as I found the concept of transcribing historic menus interesting. The visual aid they chose to use initially when accessing the site is simplistic, just featuring small close ups of photos of the menus that are being transcribed. As @Samantha-Mcguigan mentioned the photos are slightly small until you actually click on a particular menu and I agree that larger photos would be beneficial for really being able to get a glimpse into what you are about to click on. I do think that the addition of the small tab over the menu photos stating whether the menu is done or still under review is a very helpful feature for users though, especially since the site is encouraging users to possibly help review. When you do click on a menu, the photos of the original menus are a nice touch. I find the little scroll bar on the side that plainly states all the dishes and prices to be the most helpful aid when looking into the menus as it is quick and easy to scroll through. One visual aid the site had that I thought was very interesting was the Map Our Menus area, where the site used a geotagger tool to allow users to type in specific locations and have menus from around those areas pop up. Being able to type in a very specific address or just click on a general area on the map and have a corresponding menu pop up is very engaging and is a great visual tool to really put importance on the location of the menus. It is a very cool addition to the site, but I did find it a little difficult to find on the site as it is placed on the side on the main home page. So, perhaps featuring it better on the site, like near the top by the other pages on the site, would offer more attention to that feature. 

An area of the site that I investigated to determine the effectiveness of its navigation for users is the Dishes section that includes all the dishes that are on the various menus that are transcribed on the site. The concept of this portion of the site is really fascinating as it shows how many menus each type of particular dish is featured on, but the layout is a bit plain and could possibly benefit from having more ways to sort the dishes. The dishes can be sorted by date, in alphabetical order, by popularity of the dish, or by how obscure the dish is, but especially when considering the menus are from all over the world, I believe another helpful way for organization could be organizing the dishes by location. Also, having a better display of the particular dish could be helpful as the only indication of what dish you are clicking on is just by the title and the photo of a white plate that is under every dish name that includes how many menus that dish is on. Maybe including a photo of the menu that dish comes from or some of the menus could add some more visual clarity to matching up dishes with the menus before you actually click on the dish. The timeline feature on the top is very effective in finding dishes from particular eras though and works very well with finding specific dishes from that time as it also allows the user to click on a specific date and also sort the dishes from that time by the four options they have available. The various options they do have now to already search through all the dishes does make looking for specific dishes convenient, and I especially found the popularity and name sorting to be very helpful.

**Comment by gretchenuhrinek** [comment]
Being a huge Tolkien nerd, I immediately chose to explore the LotR Project. The site wouldn't make a lick of sense to someone uninitiated with LotR, but for me it was amazing. There are so many pages! The main page shows a family tree of characters from the series, but it has a few flaws. For one, there's a whole lot of empty space on the screen if you move left, right, or down -- to the point where the user may feel that she's messed up somewhere along the line, or her computer isn't loading the page properly. It's easy to get lost in the white space. Another problem is that users can't even see the top of the tree, so there's no way to know which family line you're following unless you have Stephen Colbert-levels of Tolkien lore stored in your brainspace. The legend for the family tree is basically hidden, and I only found it by accident when trying to escape the white void of empty space all around it. It would help quite a lot if users could zoom out of the tree so they could have a better idea of where they're looking. Overall, the family tree was the most annoying and least impressive aspect of the site. I don't know why it's on the homepage.

Upon exploring the rest of the site, I'm impressed by the map of Middle Earth. Locations are marked with red dots of varying sizes (perhaps to show importance?), and upon clicking on a dot a bubble appears that lists important events that happened there. On the separate Timelines page, this same map is showcased, but alongside it is an actual timeline of Middle Earth history. This is very effective, especially because when a user clicks on an event on the timeline, a line appears connecting it to the location on the map. I like that the Map and Timeline ultimately contain the same information, but are laid out in such differing ways that a researcher could easily find what she needs. I also loved the statistics page, as it was aesthetically pleasing, user-friendly, highly informative, and showcased how beneficial digital markup is to the humanities. I wish that there was a "back to top" option after each statistical entry, though. 

There are other pages that may be helpful for researches, and provide yet more examples of how markup can be used in the humanities. The Sentiment Analysis page includes horizontal graphs that show an analysis of the feelings depicted on each page in five of Tolkien's most popular works. According to the page, "Green, yellow, and red indicate positive, neutral, and negative sentiments respectively." It's interesting to see that his books are primarily neutral, and the Hobbit seems to have more negative sentiments than the other four works combined. However, I would have liked to been able to view each work individually, and an actual legend (instead of a written explanation of the color coding) would be useful.

**Comment by ajnewton1** [comment]
I was actually going to look into "What's on the Menu" as well but instead I went onto "ORBIS" to try to figure out what it actually it was. Before actually entering the site, a pop-up menu come up and basically describes what the site is intended to do and how they did it. The 'Introduction' tab on this pop-up menu displays various visualizations that describe why they used it to calculate travel distance, the cost to ship, and how long it would take to travel the distance. The maps and graphs they use are very neat and user appealing. There are many other unique tabs to explore in this pop-up menu but I didn't go through all of them. I did go in some others which are the 'Buildings' tab which showed graphs of the building process of calculations to the time it took to travel and also the 'Geospatial' tab which showed how they used GIS technology to advance their research. If you are interested in the maps they use then I would suggest looking at the 'Gallery' tab because it shows some pretty maps and some outstanding graphs that they implemented. To get out of the pop-up menu, just click the little 'X' on the top right or just click anywhere outside the pop-up.

They actual site takes some time to get used to but once you get the hang of it its pretty neat. On the left hand side of the page, there are different options for display which include terrains, sites, names, regions, paths, and all sites. The terrains box displays the terrain and outline of the the Roman Empire. I would highly recommend clicking this box in order to fully be able to see what all is going on. The sites box puts little dots on the major places to show where they were located. The names box displays the names of the sites that are mapped out on the map. The paths box shows the routes of road, river, coastal sea, and open sea. You can limit the routes being shown by clicking the options on the right side under 'Network Modes'. The regions box outlines the regions of the Empire. The all sites box displays every single site that was used for transportation. By checking both the 'all sites' box and the 'names' box can really clutter your map and make it extremely hard to read. On the left side, you can calculate the route, network, and flow of the routes. You can also calculate the route from a picked place and to a picked place meaning that the user can pick the destinations for the calculations. You can also pick different departing seasons as well as months which I thought was interesting. Another neat feature is being able to pick a different priority for the route. So instead of always looking at the fasting route, you can choose from the cheapest or even the shortest. My favorite feature about this site is being able to pick the different modes of transportation so you can choose from donkey or wagon or even civilian or military. All of these features that I described above are all correlated onto the map for visualization or even some make there own map which is awesome to look at. Overall, I like this site a lot and I am impressed with the work they put into making this.

**Comment by ajnewton1** [comment]
@gretchenuhrinek I looked into this site and had no idea what was going on. Props to you for being a "huge Tolkien nerd" because I would never be able to do that. I agree that the main page for the family tree is confusing as heck. I only managed to go all the way to the right of the page until I exited out because I didn't feel like dealing with all the 'white space'. Upon some more investigation on the site. I stubbled upon the 'Maps' page and noticed that it has little similarity to the "ORBIS" map but I grew very fond to it. It was a simplistic map but it made the point very clear and drew a direct line to the destination without having anything bother it. I liked the map a lot.

**Comment by jonhoranic** [comment]
Had to go with the LOTR: Project, that is a no-brainer for me. The whole thing just screams Tolkien nerd eye-candy, and it does not disappoint!

Scanning through the site I am amazed by the level of dedication to in depth study that had to be done just to compile all the information, let alone mark it up AND THEN create visualizations. Honestly, the amount of work that had to go into this is staggering, and I am even more surprised how versatile and polished the site is. Each page is easy to navigate to/from and they are all well explained. If I had one UI gripe it would be that some of the zoom in/out functions are buggy or are non existent in some areas where I would reasonably expect them to appear (like the massive family tree that is easy to get a bit lost in).

The map section though is great! I love the overlays and the ability to see specific sites with clickable information in pop up text boxes. I feel that a reader or fan of Tolkien's work could use this as a partner piece that could enhance the understanding of the massive amount of world building inside of lands of Middle Earth. Study wise all the statistics page has fascinating charts and graphs that are well explained and are fascinating to see because it shows the amazing level of consistency Tolkien had while building up his fictional world over the years.

In short, a grade A effort that could use some polish here or there. I love this site, and noticing that they have a store? I know they will be getting a nice "donation" from me so I can have yet another nerdy poster (or two) in the house!

**Comment by ahunker** [comment]
I too looked at the LOTR: Project because I'm a fan of Tolkien. The amount of in-depth research gone into this project is insane. The family tree alone is a huge undertaking and creating that large of a visual that has so many connections is very impressive. There are a few issues with the timeline, though. As @gretchenuhrinek mentioned above, there was a lot of whitespaces and I didn't know there was even a legend until I read her review. If there was a way to present it with the legend moving along with you as you navigate the page I think that would be easier. As I was looking around I was wishing there was a search box and that's when I found that and different ways to navigate the family tree at the top of the page. While the search box was a good idea, it doesn't quite work right and the different options for navigation and highlights are good but it's hard to view all the information at once. 
 The more I explored the site, the more engaged and impressed I was. I liked the interactive map. It was really interesting to see the different paths and place and timelines visually represented on the one map. The amount of information recorded on this map alone is insane. I applaud those involved in this project and the dedication they have to this project. The one small thing I would have liked for the interactive map has to do with the paths. I just wish that when you clicked on different names to see their paths, that the names were color coded with the paths. I wanted to compare different paths but it was hard to keep track of who was which path. 
 Overall, I really like this site and I'll definitely be returning when I reread LOTR and The Hobbit. It will be interesting to see what other information they choose to present visually.

**Comment by ebeshero** [comment]
@Samantha-Mcguigan @bsf15 @gretchenuhrinek @ajnewton1 @ahunker @jonhoranic Really excellent discussion UX from everyone here! This is wonderful: you've all taken an interest in the layout and design of these sites, as they guide your eye to orient you to complicated information. I especially appreciate all the attention to mapping here on each project!

That LOTR site is amazingly complex and intricate, and like you all I tend to be overwhelmed by his geneaology charts and sentiment analysis. To me, without a doubt, the best feature of that project is the integration of a map with a timeline, so on click of a point on the map, you get information about events in the Tolkien books that occurred on that site. 

You all might find wonderful (like I do) that the LOTR project is the work of a college student (a Chemical Engineering student who must love Tolkien a LOT) at the University of Gothenberg, Sweden. I think he made this site entirely on his own for fun(!) See more and meet the designer here: http://lotrproject.com/about/ 

@bsf15 @Samantha-Mcguigan @gretchenuhrinek @ajnewton1 @ahunker @jonhoranic @RJP43
